octoprint_discordremote/discord.py

```diff
@@ -280,6 +280,10 @@ def handle_dispatch(self, js):
         if self.me != None and user == self.me:
             # Don't respond to ourself.
             return
+        
+        if data['author'].get("bot", False):
```

Comment:
you have this backwards, your bailing out if the user is not a bot :/

Reply:
(Ideally, i would like a unit test, but given I never created one for this function either, you dont need to)

Reply:
I didn't understand.

`Message.author.bot` is `True` when the author is bot. (docs)[https://discord.com/developers/docs/resources/user#user-object]
And, it is a optional field, so I set default value to `False`.

If the value is `True`, It will return not to respond to bots.

I may have misinterpreted (this sentence)[https://github.com/cameroncros/OctoPrint-DiscordRemote/pull/163/files/3bdad6fd47d2eb6ab310730df3bbbabb465c6964#r527753360] because English is not my first language :(

Reply:
Sorry, you are correct, I misread your code.